### PR TITLE
Release Notes for TELCODOCS-1252 and TELCODOCS-1456.

### PR DESCRIPTION
**Pull request by johnwilkins**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/TELCODOCS-1456 and https://issues.redhat.com/browse/TELCODOCS-1252

Link to docs preview: http://184.23.213.161:8080/TELCODOCS-1456-rn-4.14/release_notes/ocp-4-14-release-notes.html#ocp-4-14-storage

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -279,6 +279,16 @@ For more information, see xref:../networking/hardware_networks/about-sriov.adoc#
 [id="ocp-4-14-storage"]
 === Storage
 
+[id="ocp-4-14-storage-device-selector"]
+==== Support for OR logic in LVMS
```
Comment (dfitzmau) [inline release_notes/ocp-4-14-release-notes.adoc:283]: Spell out LVMS and LVM on first use?

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -279,6 +279,16 @@ For more information, see xref:../networking/hardware_networks/about-sriov.adoc#
...
+
+With this release, the LVM Cluster Custom Resource (CR) provides `OR` logic in the `deviceSelector` section. In previous releases, specifying the `paths` setting for device paths used `AND` logic only. With this release, you can also specify `optionalPaths`, which supports `OR` logic. For more information, see the CR examples in xref:../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#persistent-storage-using-lvms[Persistent storage using logical volume manager storage].
+
+[id="ocp-4-14-storage-lvms-ext4-support"]
+==== Support for ext4 in LVMS
+
+With this release, the LVM Cluster Custom Resource (CR) provides support for the `ext4` filesystem with the `fstype` setting under `deviceClasses`. The default filesystem is `xfs`. For more information, see the CR examples in xref:../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#persistent-storage-using-lvms[Persistent storage using logical volume manager storage].
```
Comment (dfitzmau) [inline release_notes/ocp-4-14-release-notes.adoc:290]: "LVM Cluster Custom Resource (CR)"

I'm unsure why "Cluster" is capitalized? According to the SSG, "custom resource" is the correct case:

Screenshot from 2023-09-27 09-49-28

Comment (dfitzmau) [reply release_notes/ocp-4-14-release-notes.adoc:290]: `deviceClasses` subdirectory?

**Review by dfitzmau — COMMENTED** [review]
/remove-label peer-review-in-progress

/label peer-review-done

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (johnwilkins) [reply release_notes/ocp-4-14-release-notes.adoc:283]: Done.

File: release_notes/ocp-4-14-release-notes.adoc (hunk 2 above)
Comment (johnwilkins) [reply release_notes/ocp-4-14-release-notes.adoc:290]: Done

**Pull request merged**